src/assets/_variables.scss

```diff
@@ -18,19 +20,19 @@ $mediumGrey: #606266;
 $lightGrey: #909399;
```

Comment:
Change this to #F8F8F8 then change background to $lighGrey to avoid confusion in the future.